org-roam.el

```diff
@@ -113,6 +113,11 @@ If nil, `find-file' is used."
   :type 'function
   :group 'org-roam)
 
+(defcustom org-roam-update-db-idle-delay 2
```

Comment:
rename to `org-roam-update-db-idle-seconds`, to make it more explicit that it is the number of seconds of idle time.